### PR TITLE
Add a blog for 12.07

### PR DESCRIPTION
**Pull request by HashidaTKS**
Add a blog for 12.07

### CODE REVIEW

**Comment by HashidaTKS** [comment]
@komainu8 

Would you review this?

File: en/_posts/2022-08-30-mroonga-12.07.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+* How to install: [Install](/docs/install.html)
+* How to upgrade: [Upgrade guide](/docs/upgrade.html)
+
+### Changes
+
+The main changes are as follows.
+
+### Improvements
+
```
Comment (komainu8) [inline en/_posts/2022-08-30-mroonga-12.07.md:23]:
```suggestion
```

File: en/_posts/2022-08-30-mroonga-12.07.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* Fixed a bug to fail to install or upgrade Mroonga with install.sql.
+  
+  Broken install.sql caused this bug. This bug had occurred since Mroonga 12.06.
+
+* Fixed a bug that `mroonga_command()` returns results with hex dump for MySQL 8.0 and above.
+
+  `mroonga_command()` returns results with a correct character code form this version.
+
+* [AlmaLinux](/docs/install/almalinux.html) Fixed an installation document for Mroonga on AlmaLinux 8.
+
```
Comment (komainu8) [inline en/_posts/2022-08-30-mroonga-12.07.md:46]:
```suggestion
```

File: ja/_posts/2022-08-30-mroonga-12.07.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+* インストール方法: [インストール](/ja/docs/install.html)
+* アップグレード方法: [アップグレードガイド](/ja/docs/upgrade.html)
+
+### 変更点
+
+今回の変更点は以下の通りです。
+
+### 改良
+
+* [CentOS](/docs/install/centos.html), [AlmaLinux](/docs/install/almalinux.html) MariaDB 10.3.36, 10.4.26, 10.5.17, 10.6.9, 10.7.5 and 10.8.4をサポートしました。
```
Comment (komainu8) [inline ja/_posts/2022-08-30-mroonga-12.07.md:20]:
```suggestion
* [CentOS](/ja/docs/install/centos.html), [AlmaLinux](/ja/docs/install/almalinux.html) MariaDB 10.3.36, 10.4.26, 10.5.17, 10.6.9, 10.7.5 and 10.8.4をサポートしました。
```

File: ja/_posts/2022-08-30-mroonga-12.07.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+* [CentOS](/docs/install/centos.html), [AlmaLinux](/docs/install/almalinux.html) Percona Server 5.7.39-42をサポートしました。
```
Comment (komainu8) [inline ja/_posts/2022-08-30-mroonga-12.07.md:22]:
```suggestion
* [CentOS](/ja/docs/install/centos.html), [AlmaLinux](/ja/docs/install/almalinux.html) Percona Server 5.7.39-42をサポートしました。
```

File: ja/_posts/2022-08-30-mroonga-12.07.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+* [CentOS](/docs/install/centos.html), [AlmaLinux](/docs/install/almalinux.html) Percona Server 8.0.29-21をサポートしました。
```
Comment (komainu8) [inline ja/_posts/2022-08-30-mroonga-12.07.md:24]:
```suggestion
* [CentOS](/ja/docs/install/centos.html), [AlmaLinux](/ja/docs/install/almalinux.html) Percona Server 8.0.29-21をサポートしました。
```

File: ja/_posts/2022-08-30-mroonga-12.07.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+* install.sqlを使用したインストールおよびアップグレードに失敗する問題を修正しました。
+
+  install.sqlが壊れていたことが原因です。この問題は、 Mroonga 12.06以降で発生していました。
+
+* MySQL 8.0 以上を使用しているとき、`mroonga_command()` が16進数ダンプで結果を返す問題を修正しました。
+
+  このバージョンから、``mroonga_command()`` が正しい文字コードで結果を返すようになります。
+
+* [AlmaLinux](/docs/install/almalinux.html) AlmaLinux 8 向けMroongaのインストール手順を修正しました。
```
Comment (komainu8) [inline ja/_posts/2022-08-30-mroonga-12.07.md:40]:
```suggestion
* [AlmaLinux](/ja/docs/install/almalinux.html) AlmaLinux 8 向けMroongaのインストール手順を修正しました。
```

**Review by komainu8 — CHANGES_REQUESTED** [review]
Could you confirm my comments?

**Pull request merged**